src/templates/signup-block/form-signup-block.vue

```diff
@@ -1,204 +1,121 @@
 <template>
-  <div class="w-full flex flex-col gap-8 animate-fadeIn">
-    <form
-      class="flex flex-col gap-8"
-      @submit.prevent="signUp"
-    >
-      <div class="flex flex-col gap-2">
-        <label
-          for="email"
-          class="font-semibold text-sm"
-          >Email</label
-        >
-        <InputText
-          v-model="email"
-          id="email"
-          type="email"
-          class="w-full"
-          :class="{ 'p-invalid': errors.email }"
-        />
-        <small
-          v-if="errors.email"
-          class="p-error text-xs font-normal leading-tight"
-          >{{ errors.email }}</small
-        >
-      </div>
-      <div class="flex flex-col gap-2">
-        <label
-          for="name"
-          class="font-semibold text-sm"
-          >Full Name</label
-        >
-        <InputText
-          v-model="name"
-          id="name"
-          type="text"
-          class="w-full"
-          :class="{ 'p-invalid': errors.name }"
-        />
-        <small
-          v-if="errors.name"
-          class="p-error text-xs font-normal leading-tight"
-          >{{ errors.name }}</small
-        >
-      </div>
-      <div class="flex flex-col gap-2">
-        <label
-          for="password"
-          class="font-semibold text-sm"
-          >Password</label
-        >
-        <InputPassword
-          toggleMask
-          v-model="password"
-          id="password"
-          class="w-full"
-          :class="{ 'p-invalid': errors.password }"
-          :feedback="false"
-        />
-        <small class="p-error text-xs font-normal leading-tight">{{ errors.password }}</small>
-
-        <label class="font-semibold text-sm my-2">Must have at least:</label>
-        <ul class="text-color-secondary list-inside space-y-3">
-          <li
-            class="flex gap-3 items-center text-color-secondary"
-            :key="i"
-            v-for="(requirement, i) in passwordRequirementsList"
+  <div class="w-full md:min-w-[380px] md:max-w-[448px]">
```

Comment:
remove fixed values, try to talk with the design team to use only tailwind tokens instead.